controllers/common.go

```diff
@@ -1078,3 +1078,50 @@ func mergeErrors(errs []error) string {
 	}
 	return strings.Join(errStrings, " ; ")
 }
+
+func (r *MultiClusterHubReconciler) GetSubConfig() *subv1alpha1.SubscriptionConfig {
```

Comment:
This method is missing error handling. It makes calls to the API server so it's at risk of failing, but ignores errors.